deploy/examples/cluster.yaml

```
@@ -222,6 +222,8 @@ spec:
 # nodes below will be used as storage resources.  Each node's 'name' field should match their 'kubernetes.io/hostname' label.
     # nodes:
     #   - name: "172.17.4.201"
+    #     config:
+    #       metadataDevice: "/dev/disk/by-id/nvme-WDC_PC_SN730_SDBQNTY-256G-1001_1952FL441909" # metadataDevice can be specified using full udev path
```

```suggestion
    #       metadataDevice:  # metadataDevice can be specified using full udev path
```
I would suggest keeping this field empty since it is commented it will be no problem and also I don't think giving complete disk name is good here.